flutter/example/integration_test/integration_test.dart

```diff
@@ -124,4 +132,77 @@ void main() {
     final transaction = Sentry.startTransactionWithContext(context);
     await transaction.finish();
   });
+
+  group('e2e', () {
+    var output = find.byKey(const Key('output'));
+    late Fixture fixture;
+
+    setUp(() {
+      fixture = Fixture();
+    });
+
+    testWidgets('captureException', (tester) async {
+      await setupSentryAndApp(tester, dsn: exampleDsn);
+
+      await tester.tap(find.text('captureException'));
+      await tester.pumpAndSettle();
+
+      final text = output.evaluate().single.widget as Text;
+      final id = text.data!;
+
+      final uri = Uri.parse(
+        'https://sentry.io/api/0/projects/$org/$slug/events/$id/',
+      );
+
+      final event = await fixture.poll(uri, authToken);
+      expect(event, isNotNull);
+      expect(fixture.validate(event!), isTrue);
+    });
+  });
+}
+
+class Fixture {
+  Future<Map<String, dynamic>?> poll(Uri url, String authToken) async {
+    final client = Client();
+
+    const maxRetries = 10;
+    const initialDelay = Duration(seconds: 2);
+    const factor = 2;
+
+    var retries = 0;
+    var delay = initialDelay;
+
+    while (retries < maxRetries) {
+      try {
+        final response = await client.get(
+          url,
+          headers: <String, String>{'Authorization': 'Bearer $authToken'},
+        );
+        if (response.statusCode == 200) {
+          return jsonDecode(utf8.decode(response.bodyBytes));
+        }
+      } catch (e) {
+        // Do nothing
+      } finally {
+        retries++;
+        await Future.delayed(delay);
+        delay *= factor;
+      }
+    }
+    return null;
+  }
+
+  bool validate(Map<String, dynamic> event) {
```

Comment:
Can we actually validate more properties in the event?
I know this is a copy of https://github.com/getsentry/sentry-dart/blob/main/e2e_test/bin/e2e_test.dart
Maybe we should improve that test as well.
The goal is that we assert every property that is static and not dependent on the host platform, so it wont fail in different devices, etc.

Reply:
Let's start with the current approach and expand as we go.
Ideally, we'd check every property that is static, does not change depending on the device, etc.